filer/private/sass/layout/_main.scss

```diff
@@ -689,7 +689,6 @@ body {
 }
 .dropzone-upload-info {
     margin-top: 10px;
-    // make sure that file name on upload progress is cut #675
```

Comment:
actually on every change. Can you add some comments to your changes as well with pull request id?